data/mappings/keyboard_aliases.hjson

```diff
@@ -2100,6 +2106,12 @@
     "rookiebwoy/neopad": {
         "target": "rookiebwoy/neopad/rev1"
```

Comment:
The targets here also need to be updated from `rookiebwoy/*/rev1` to `ivndbt/*/rev1"`

Reply:
Got it! Thanks for your patience.